src/platforms/android/common/performance/included-instrumentation.mdx

```diff
@@ -0,0 +1,109 @@
+---
+title: Included Instrumentation
+sidebar_order: 21
+description: "Learn what transactions are captured after tracing is enabled."
+---
+
+<Note>
+
+Capturing transactions requires that you first <PlatformLink to="/performance/">set up performance monitoring</PlatformLink> if you haven't already.
+
+Supported in Sentry's Android SDK version `4.3.0` and above.
+
+</Note>
+
+### Android's Activity Instrumentation
+
+The Activity's instrumentation, once enabled, captures transactions for each launch of an Activity. The SDK sets the Transaction name to the name of the Activity, for example, `MainActivity`, and Transaction operation to `navigation`.
+
+The transaction starts before each Activity's `onCreate` method is called.
+
+_The Activity's instrumentation is enabled by default, but you may disable it by setting:_
+
+```xml {filename:AndroidManifest.xml}
+<manifest>
+    <meta-data android:name="io.sentry.traces.activity.enable" android:value="false" />
+</manifest>
+```
+
+The transaction finishes after each Activity's `onResume` method is executed.
+
+_The transaction finishes automatically, but you may disable it by setting:_
+
+```xml {filename:AndroidManifest.xml}
+<manifest>
+    <meta-data android:name="io.sentry.traces.activity.auto-finish.enable" android:value="false" />
+</manifest>
+```
+
+We offer the possibility to finish the transaction manually. For example, you might want to finish the transaction after an API call triggered on Activity's `onCreate` and shown the data to the user. To achieve that you can do:
+
+```java
+import io.sentry.Sentry;
+
+ISpan span = Sentry.getSpan();
+if (span != null) {
+  span.finish();
+}
+```
+
+```kotlin
+import io.sentry.Sentry
+
+val span = Sentry.getSpan()
+span?.finish()
+```
+
+In case that the user left this Activity before you've finished the transaction manually, the SDK finishes the transaction automatically when the `onDestroy` method is called.
+
+Transactions are always bound to the Scope automatically if there's none set. Because of that, you may create spans using manual instrumentation, and those spans will be automatically associated with the Activity's running transaction.
+
+```java
+import java.lang.Exception;
+
+import io.sentry.ISpan;
+import io.sentry.Sentry;
+import io.sentry.SpanStatus;
+
+void displayUserData() {
+  ISpan span = Sentry.getSpan();
+  if (span != null) {
+    ISpan innerSpan = span.startChild("displayUserData");
+    try {
+      // omitted code
+    } catch (Exception e) {
+      innerSpan.setThrowable(e);
+      innerSpan.setStatus(SpanStatus.NOT_FOUND);
```

Comment:
Curious if we need a status when an exception is set. Is it optional?
In case there's no status and an exception is set, can we infer the status via the exception type?

https://github.com/getsentry/sentry-dotnet/blob/64d4b36002e179464231189f709e98dee087d822/src/Sentry/Span.cs#L95-L110

/cc @maciejwalkowiak